### PR TITLE
Fixed Issue #466 by simply changing the default versionName to return "1.0" instead of null in gradle script if none of the OS matches

### LINKED ISSUE #466: java Null Pointer Exception on getVersionedAppName() method thus causing app to crash
#### Software and hardware versions 
Collect v1.x.x, Android v5.1.1, device used...

#### Problem description
App Crashes as soon as it installs.

#### Other information 
Things you tried, stack traces, related issues, suggestions on how to fix it...

### PR DESCRIPTION
**Pull request by thegenuinegourav**

### CODE REVIEW

**Comment by yadav-rahul** [comment]
@thegenuinegourav It had be better if you make new branch before submitting PR. Please have a look at it here https://github.com/opendatakit/collect/blob/master/CONTRIBUTING.md#submitting-a-pull-request.

File: collect_app/src/main/java/org/odk/collect/android/application/Collect.java [hunk 1]
```diff
@@ -45,6 +46,8 @@
 
 import java.io.File;
 
+import static java.security.AccessController.getContext;
+
```
Comment (yadav-rahul) [inline collect_app/src/main/java/org/odk/collect/android/application/Collect.java:50]: Why this extra space?

File: collect_app/build.gradle [hunk 2]
```diff
@@ -52,18 +52,18 @@ def getVersionName = { ->
...
     } catch (ignored) {
-        return null;
+        return "1.0"; //Returning 1.0 by default
     }
 }
 
 android {
-    compileSdkVersion(22)
-    buildToolsVersion("25.0.0")
+    compileSdkVersion 25
```
Comment (yadav-rahul) [inline collect_app/build.gradle:60]: @lognaturel @yanokwa Should we change `compileSdkVersion` to 25?

**Comment by lognaturel** [comment]
Please do not mix different changes within the same PR. Having a focused PR makes it easier to review and less likely to introduce errors. If you'd like to do some refactoring or change versions, please do those things in separate PRs. As I mentioned before, we are not updating sdk versions at this time.

This fix is not appropriate either because there should always be a valid version to show. If you need help finding the source of the NPE, please update your issue with the information I requested.

**Pull request closed without merging**